kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,161 @@
+local influxdb_serializer = require "kong.plugins.log-serializers.influxdb"
```

Comment:
You build on top of the commit to re-use the http-log code, but then you do not actually reuse it... Can you base it of the http-log and remove the duplicate code?